config/initializers/fog_overrides.rb

```diff
@@ -0,0 +1,19 @@
+require 'fog/vsphere/requests/compute/get_virtual_machine'
```

Comment:
Isn't this going to fail if Fog isn't installed?  Can we use existing fog extensions?